PRS7001-8000/PRS7680Os.xml

```diff
@@ -45,19 +48,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
          <change who="PL" when="2019-03-27">removed occupation containing St, Saint, bibl, biblical or prophet, added keyword. See https://github.com/BetaMasaheft/Documentation/issues/1104</change>
+         <change who="MKr" when="2023-06-13">Added bibliography, floruit</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  ʿOṣ
-               </persName>
+               <persName xml:lang="gez" xml:id="n1">ዖጽ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">ʿOṣ</persName>
                <!-- St -->
+  <floruit>His name appears in some lists of the <ref type="pers" corresp="PRS12196NineSaints">Nine Saints</ref>, usually substituting for <ref type="pers" corresp="PRS8711sehmas">Ṣǝḥmā</ref>. The place of his origin is Kʷǝzārā, possibly a corruption
```

Comment:
I would suggest to speak about a "transcription" rather than "corruption" - evem though it is a very odd way of transcribing Caesarea/Καισάρεια/قيسارية . Another option might be to think about الخزر Khazars, but this record is not the right place to discuss several options in parallel.